controllers/custom/custom_controller.go

```diff
@@ -221,8 +239,15 @@ func (c *CustomController) processNextWorkItem() bool {
 
 func (c *CustomController) reconcileHandler(obj interface{}) bool {
 	var req Request
+	var normalRequest reconcile.Request
 	var ok bool
-	if req, ok = obj.(Request); !ok {
+
+	// Requests generated through a watch return reconcile.Request
+	// instead of custom.Request, so we must convert it to custom.Request
+	if normalRequest, ok = obj.(reconcile.Request); ok {
```

Comment:
I'm not thrilled with this approach, but its required unless a custom Event Handler is created which returns a custom.Request instead.